### PR TITLE
[Stats Refresh] Add card template with tabbed views

### LINKED ISSUE #10635: [Stats Refresh] Add card template with tabbed views
Add a view template with tabs to be used by cards like Insights `Comments` and `Followers`.

- 2 tabs.
- `View more` option.
- `No data yet` for each tab.

Example designs:
comments
followers

### PR DESCRIPTION
**Pull request by ScoutHarris**
Fixes #10635 

This adds a card template that will be used for cards with tabbed data (i.e. Insights Comments and Insights Followers).

View components include:
- `FilterTabBar` to switch between data views.
- Total count.
- Subtitles.
  - Subtitles and total count hide if there is no data.
- Data rows.
  - The disclosure does nothing yet.
- `View more` if the number of data rows exceeds 6.
  - This shows a temporary alert.
- `No data yet` if there is no data.

To test:
- Go to Stats > Insights.
- Scroll to the Comments card. Since this is just a template, the data shown is purely fake.
- Switch between the tabs. Verify the views update accordingly.

tabs

---
To see the different states, you can modify the parameters sent when creating the data for the card. It's all done in `SiteStatsInsightsViewModel:createTabbedTotalsStatsRow`.

To hide the total count label, modify the `TabbedTotalsStatsRow:showTotalCount` parameter.

no_total_count

To see `No data yet`,  send an empty array for the `TabData:dataRows` parameter.

no_data

### CODE REVIEW

**Comment by ScoutHarris** [comment]
@jklausa - to see the constraint warnings we talked about, it happens when showing the Total Count label, and one of the tabs has no data. So:
- Send no data rows for one of the tabs.
- Set `showTotalCount` to `true`.

**Comment by ScoutHarris** [comment]
@jklausa - also, I noticed on rotation sometimes the hidden bits become visible. It's happening on `develop`, so I broke it long before now. I'll fix it in a separate PR.

hidden_shown

File: WordPress/Classes/ViewRelated/Stats/Insights/SiteStatsInsightsTableViewController.swift [hunk 1]
```diff
@@ -66,6 +67,7 @@ class SiteStatsInsightsTableViewController: UITableViewController {
         ImmuTable.registerRows(tableRowTypes(), tableView: tableView)
         loadInsightsFromUserDefaults()
         initViewModel()
+        tableView.estimatedRowHeight = 500
```
Comment (jklausa) [inline WordPress/Classes/ViewRelated/Stats/Insights/SiteStatsInsightsTableViewController.swift]: 🤔 Why 500 ;P? Is this needed for performance purposed or just any non-zero value to kick off some changes in how UITableView lays itself out?

Comment (ScoutHarris) [reply WordPress/Classes/ViewRelated/Stats/Insights/SiteStatsInsightsTableViewController.swift]: Oh sorry! I should have commented or something. That's related to [this](https://github.com/wordpress-mobile/WordPress-iOS/pull/10649/commits/870d5a6029257e2eaa83b34ba70a85ad130d22d8) change. Basically, `FilterTabBar` was throwing constraint warnings without `estimatedRowHeight` set. The `TabbedTotalsCell` generally has a height of 400-ish (which thus far is the highest of all cells), so I set it at 500.

Comment (jklausa) [reply WordPress/Classes/ViewRelated/Stats/Insights/SiteStatsInsightsTableViewController.swift]: 👍

**Comment by ScoutHarris** [comment]
To summarize a Slack discussion, there's still constraint warnings in one particular scenario. Since I'll be modifying the xib for Followers (and maybe Comments, who knows), I'll make an attempt to fix it in a follow up PR.

@jklausa thank you so much for your time and fixing at least some of the warnings! 😄

**Pull request merged**